data-prepper-plugins/common/src/test/java/com/amazon/dataprepper/plugins/prepper/state/PrepperStateTest.java

```diff
@@ -33,6 +33,7 @@ public void testSize() {
         prepperState.put(key2, data2);
 
         Assert.assertEquals(2, prepperState.size());
+        Assert.assertEquals(2097152, prepperState.sizeInBytes());
```

Comment:
This will not work when you implement the size function for lmdb, because sizes will not be the same. For example, MapDB allocates file size in MB increments, which is why this is exactly 2MB.